lib/highline.rb

```diff
@@ -677,20 +449,17 @@ def indentation
   def indent(increase=1, statement=nil, multiline=nil)
     @indent_level += increase
     multi = @multi_indent
-    @multi_indent = multiline unless multiline.nil?
+    @multi_indent = multiline || @multi_indent
```

Comment:
You could do `@multi_indent ||= multiline`. Its more cleaner and easy to read. =)